proto/axelar/multisig/v1beta1/events.proto

```diff
@@ -15,6 +15,10 @@ message KeygenStarted {
   repeated bytes participants = 3
       [ (gogoproto.casttype) =
             "github.com/cosmos/cosmos-sdk/types.ValAddress" ];
+  repeated bytes weights = 4 [
```

Comment:
I don't like relying on the order to match element from here to participants, but it's ok for now.